src/components/EmailInviteConfirmation/stylesheet.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../variables';
+
+body {
+  background-color: $theme-dark-background;
+  color: $theme-dark-foreground;
+}
+
+.EmailInviteConfirmation {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 160px;
+  padding-bottom: 38px;
+  padding-left: 32px;
+  padding-right: 32px;
+  font-size: 24px;
+  height: 100%;
+
+  @media (max-width: 1000px) {
```

Comment:
The page is responsive to the width but not the height. The button appears below the BoG text and logo:

<img src="https://user-images.githubusercontent.com/57265280/230686266-d850f99c-a443-4183-a9aa-a5a465c4a597.jpg" width=200 height=450 />